.circleci/config.yml

```diff
@@ -17,22 +17,11 @@ workflows:
             tags:
               only: /^v.*/
 
-      - architect/integration-test:
-          context: architect
-          name: basic-integration-test
-          test-dir: "integration/test/basic"
-          filters:
-            # Do not trigger the job on merge to master.
-            branches:
-              ignore:
-                - master
-          requires:
-            - push-to-default-app-catalog
-
       - architect/integration-test:
           context: architect
           name: metrics-integration-test
-          test-dir: "integration/test/basic"
+          setup-script: "integration/test/metrics/setup.sh"
```

Comment:
The setup script installs apptestctl. This will move to a new param on the integration-test job.